resources/assets/components/ModalContainer/modal-container.scss

```diff
@@ -0,0 +1,104 @@
+@import '~@dosomething/forge/scss/toolkit';
+
+$tablet: "(min-width: 760px)" !default;
+$base-spacing: 24px !default;
+$light-gray: #ddd !default;
+
+.modal-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  min-height: 100%;
+  z-index: 9999;
+  background: rgba(#000, 0.5);
+
+  @media #{$tablet} {
+    padding: 72px;
+  }
+
+  // Animations (such as `fadeOut` and `moveDown`) are defined in Neue
+  // @TODO: Is there a reasonable way to share these?
```

Comment:
@sbsmith86 this is actually copied over from DS/Modals and we aren't using this part right now anyway. I wasn't sureee if we wanted to include it anyway though.